From 7e7c86cceb2582bba96a36f217a3b8199c80eb1d Mon Sep 17 00:00:00 2001
From: Hanyuan Zhao <hanyuan-z@qq.com>
Date: Fri, 16 Feb 2024 17:37:31 +0800
Subject: [PATCH 08/12] llep: set FW_JUMP_ADDR to 0x0300_0000

Signed-off-by: Hanyuan Zhao <hanyuan-z@qq.com>
---
 platform/llep/objects.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platform/llep/objects.mk b/platform/llep/objects.mk
index 7216356..2cf9072 100644
--- a/platform/llep/objects.mk
+++ b/platform/llep/objects.mk
@@ -53,4 +53,4 @@ PLATFORM_RISCV_ISA=rv32ima_zicsr_zifencei
 #
 FW_JUMP=y
 # This needs to be 4MB aligned for 32-bit support
-FW_JUMP_ADDR=0x02400000
+FW_JUMP_ADDR=0x03000000
-- 
2.34.1

